### DISCUSSION TITLE
Is there any way to use .po files instead of .json files for translations?

### DISCUSSION THREAD

**Discussion by juagarc4** [vinissimus/next-translate#544] [Q&A] [answered]
**Problem/Motivation:**

Many translations are done by external translators that usually work with POEditors to create the translations and save them into the corresponding .po files. .po extension is very popular and useful to manipulate translation files.

**Desired solution:**
I would like to have the possibility to use .po files in the same way I'm using json files.

**Additional context**
I've tried to use the module "https://github.com/openculinary/i18next-gettext-loader" but this plugin seems to be more like a node tool to convert po files to json files than a plugin to add on the fly .po files compatibility. 

Thanks in advance.
Regards

**Comment by aralroca** [comment, marked as answer]
Yes. You need a webpack loader to load .po files. Then, instead of `locales/en/common.json` you can use `locales/en/common.po`, or using a different structure defining the `loadLocalesFrom` config prop.
